### ISSUE TITLE
MMS not downloading pictures

### ISSUE THREAD

**Issue by nicm1515** [phhusson/treble_experimentations#1132]
on UMIDigi One Max, on any 10 version of this ROM, the MMS pictures do not download. I have tired different SMS apps, checked, deleted, readded the APN settings, (including editing the apns-config files, (in both locations)) and still no downloads.  Help...

**Comment by phhusson** [comment]
Have you tried QKSMS app?

**Comment by nicm1515** [comment]
Yes, and several others. Was adviced to check the apns-conf files, but apart from no leading 0's in the MMS proxy settings, the settings are as 2dregrees settings. Have confirmed apn settings from other phones and they are the same....  Got me stuffed😁

Cheers
Nic.

**Comment by phhusson** [comment]
Have you checked the various settings in QKSMS?
It has various settings to overcome such issues

**Comment by nicm1515** [comment]
I thought I had, but any pointers to the settings to play with much appreciated.

**Comment by os54656** [comment]
Did you make sure that your Access Point Names correct? When I installed the ROM, my APN settings were screwed up

**Comment by nicm1515** [comment]
oh yeah I did, I must have spent hours on this, removing APN's, editing the APN config.xml files, trying different settings in the APN's, trying different SMS apps, nothing worked, even completely reset then, reflashed the ROM (although that was due to another issue, but same result), nothing worked.  In the end had to abandon this a get a new phone, which while it works (went stock Samsung), boy I miss some of the Stock features......

**Comment by os54656** [comment]
Oh wow.... I thought I'd give it a shot lol. Yeah I miss One UI.. Can't
remember my software version so I can't go back to a OneUI rom. But either
way, OneUI on a rooted phone isn't too much fun anyway
